include/seahorn/FiniteMapTransf.hh

```diff
@@ -0,0 +1,105 @@
+#pragma once
+
+#include "seahorn/Expr/Expr.hh"
+#include "seahorn/Expr/ExprOpBind.hh"
+#include "seahorn/Expr/ExprCore.hh"
+#include "seahorn/Expr/ExprOpFiniteMap.hh"
+#include "seahorn/Expr/ExprVisitor.hh"
+#include "seahorn/HornClauseDB.hh"
+#include "seahorn/HornModelConverter.hh"
+
+#include "seahorn/Support/SeaDebug.h"
+
+using namespace expr;
+using namespace expr::op;
+
+namespace seahorn {
+
+// Rewrites a finite map operation whose arguments are already rewritten
```

Comment:
It would be helpful to say that the rewriter removes finite map terms

Reply:
Added to the documentation